sdk/documentintelligence/test-resources.bicep

```diff
@@ -62,9 +62,9 @@ resource cognitiveServicesAccount 'Microsoft.CognitiveServices/accounts@2024-04-
 
 output DOCUMENT_INTELLIGENCE_ENDPOINT string = cognitiveServicesAccount.properties.endpoint
 output DOCUMENT_INTELLIGENCE_API_KEY string = listKeys(cognitiveServicesAccount.id, apiVersion).key1
-output DOCUMENT_INTELLIGENCE_TRAINING_DATA_CONTAINER_SAS_URL string = '${reference(blobResourceId, storageApiVersion).primaryEndpoints.blob}${trainingDataContainer}?${listServiceSas(blobResourceId, storageApiVersion, trainingDataSasProperties).serviceSasToken}'
+output DOCUMENT_INTELLIGENCE_TRAINING_CONTAINER_SAS_URL string = '${reference(blobResourceId, storageApiVersion).primaryEndpoints.blob}${trainingDataContainer}?${listServiceSas(blobResourceId, storageApiVersion, trainingDataSasProperties).serviceSasToken}'
```

Comment:
Could we use managed identity to access the container instead?

Reply:
I'll put that as a work item for later.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/32234